src/coreclr/src/vm/eventpipebuffermanager.cpp

```diff
@@ -168,6 +168,9 @@ EventPipeBuffer* EventPipeBufferManager::AllocateBufferForThread(EventPipeThread
         const unsigned int maxBufferSize = 1024 * 1024;
         bufferSize = Min(bufferSize, maxBufferSize);
 
+        // Make the buffer size fit into with pagesize-aligned block.
```

Comment:
I'd suggest updating the comment to answer "why does it matter that the buffersize is page-aligned?"